reports/glides_validator.livemd

```diff
@@ -22,90 +24,553 @@ Mix.install(
 )
 ```
 
-## Load Data
+## Instructions
 
 ### README
 
 This report provides an implementation of https://www.notion.so/mbta-downtown-crossing/Rethinking-Prediction-Accuracy-for-Glides-e99561127b01490689135ab6b70cd33c
 
+### Generate a report
+
+1. Click "Evaluate" on the code cell in the [**Options for Report**](#options-for-report) section.
+2. Scroll to the bottom of the code cell and choose settings for your report.
+3. Jump to the [**Results**](#results) section and click "Evaluate" on its code cell.
+4. Scroll to the bottom of the code cell.
+5. Wait around 30 sec to see results. Takes longer the first time it runs. Click "Export as CSV" to save results to your Downloads folder.
+
+If you want to generate another report with different settings, simply change the settings, scroll back down to [**Results**](#results), and click "Evaluate" again.
```

Comment:
Oh hey, I guess this answers point number 1. Reading comprehension and all that.